tests/functional_tests/test_get_discoveries.py

```diff
@@ -0,0 +1,116 @@
+import os
+import tempfile
+import unittest
+
+import pandas as pd
+
+from credentialdigger.cli import cli
+from credentialdigger.client_sqlite import SqliteClient
```

Comment:
I do not think that it is necessary. For the underlying CLI command, it does not matter if we are using SQLite or Postgres, since we are making direct calls to the `client` class.

I used the SQLite client to simply set up a mock database. 

I believe that the `get_discoveries` should be put into test when we implement UI tests, otherwise we will be having redundant tests if we implement it here.